stdlib/ctypes/__init__.pyi

```diff
@@ -5,9 +5,7 @@ from collections.abc import Callable, Iterable, Iterator, Mapping, Sequence
 from typing import Any, ClassVar, Generic, TypeVar, Union as _UnionT, overload
 from typing_extensions import TypeAlias
 
-# TODO: import these from _ctypes once it no longer breaks pytype
-RTLD_GLOBAL: int
-RTLD_LOCAL: int
+from _ctypes import RTLD_GLOBAL as RTLD_GLOBAL, RTLD_LOCAL as RTLD_LOCAL
```

Comment:
I wonder why `isort` didn't move this import near the top of the file.

Reply:
Take a look at the second commit in #8656 — there's quite a few private stdlib modules that we have in typeshed that, by default, isort doesn't recognise as being part of the stdlib